packages/mattermost-redux/src/selectors/entities/general.ts

```diff
@@ -113,3 +113,5 @@ export const isMarketplaceEnabled: (state: GlobalState) => boolean = createSelec
         return config.PluginsEnabled === 'true' && config.EnableMarketplace === 'true';
     },
 );
+
+export const getWorkTemplatesLinkedProducts = (state: GlobalState) => state.entities.worktemplates.linkedProducts;
```

Comment:
:+1: for adding a selector. I think it is worth it to create a `packages/mattermost-redux/src/selectors/entities/worktemplates.ts` file since the state is somewhat complex, will likely grow more complex over time given the potential of this feature, and there are some existing ad-hoc selectors in `components/work_templates/index.tsx` that we will eventually also want to place in a selectors file.

Reply:
agree that maybe it would be worth in the future, but for now let's keep it simple since we are not sure the future roadmap of this feature.